README.md

```diff
@@ -67,6 +71,9 @@ It is recommended to use parameters as listed above. The various `publickey` and
 The `service_provider_repository` is a repository of service providers for which you offer IdP services. The service
 configured _must_ implement the `Surfnet\SamlBundle\Entity\ServiceProviderRepository` interface.
 
+Service providers can statically be provided by configuration `remote.service_providers`. To use these configured service 
```

Comment:
Suggestion to improve grammer:

Original:
| Service providers can statically be provided by configuration `remote.service_providers`.

Suggestion:
| Service providers can be provided statically by using the `remote.service_providers` configuration option.